src/modules/users/components/TokenActivation/TokenActivationContent/TokenActivationContent.tsx

```diff
@@ -2,25 +2,40 @@ import React, { useState } from 'react';
 import { FormattedMessage, defineMessages } from 'react-intl';
 
 import { Tab, Tabs, TabList, TabPanel } from '~core/Tabs';
+import { useClaimableStakedMotionsQuery } from '~data/generated';
+
+import TokensTab, { TokensTabProps } from '../TokensTab';
+import StakesTab from '../StakesTab/StakesTab';
```

Comment:
This import can be written as `../StakesTab`

Reply:
I'll clean this up. thx